pkg/controllers/awsloadbalancercontroller/rbac.go

```diff
@@ -3,9 +3,93 @@ package awsloadbalancercontroller
 import (
 	"context"
 
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
 	albo "github.com/openshift/aws-load-balancer-operator/api/v1alpha1"
 )
 
-func (r *AWSLoadBalancerControllerReconciler) ensureClusterRoleAndBindings(ctx context.Context, controller *albo.AWSLoadBalancerController) error {
+func (r *AWSLoadBalancerControllerReconciler) ensureClusterRoleAndBindings(ctx context.Context, sa *corev1.ServiceAccount, controller *albo.AWSLoadBalancerController) error {
+	err := r.ensureClusterRoles(ctx, controller)
+	if err != nil {
+		return err
+	}
+
+	err = r.ensureClusterRoleBindings(ctx, sa, controller)
+	if err != nil {
+		return err
+	}
+
+	err = r.ensureRoles(ctx, controller)
+	if err != nil {
+		return err
+	}
+
+	err = r.ensureRoleBindings(ctx, sa, controller)
+	if err != nil {
+		return err
+	}
+
 	return nil
 }
+
+func buildClusterRole(name string, rules []rbacv1.PolicyRule) *rbacv1.ClusterRole {
```

Comment:
Could you put these _build*_ functions in the files where they are used.

Reply:
Addressed